woocommerce/woocommerce
# [testing workflows] Replace daily-e2e with nightly-checks

### Changes proposed in this Pull Request:

Since we're pushing to the `nightly` tag every day, we should use that to run our daily/nightly checks instead of a scheduled job. This will use the release checks setup, and download and install for testing the built plugins attached to the tag.

- removed the `test_daily_run.yml` workflow.
- removed the workflow_call trigger. 
- replaced the `daily-e2e` event with `nightly-checks`.

### How to test the changes in this Pull Request:

Hard to test this before merging. 
After the PR is merged, we'll have to wait for the nightly tag to be updated and check CI.
